rollup.config.js

```diff
@@ -1,7 +1,26 @@
-module.exports = {
-    input: 'examples/web-components/platform.js',
-    output: {
-        file: 'examples/web-components/bundle.js',
-        format: 'cjs'
+import typescript from '@rollup/plugin-typescript'
+import { terser } from 'rollup-plugin-terser'
+
+function config(input, filePrefix) {
+    return {
+        input,
+        output: [{
+            file: `lib/${filePrefix}.js`,
+            format: 'es',
+            sourcemap: true
+        },
+        {
+            file: `lib/${filePrefix}.min.js`,
+            format: 'es',
+            sourcemap: false,
```

Comment:
any particular reason for not having sourcemap for the min file?

Reply:
Yeah usually source maps improve debugging and no one would be debugging the minified code. That's where I was coming from.

Reply:
I thought one of the basic use cases for source maps was to debug minified code while seeing readable source code in the debugger, no?

Reply:
ok sure

Reply:
Didn't mean to push you to make a change @tedconn, I was asking if the scenario I mentioned made sense 🙂

Reply:
I think it does. ON the platform either you're in debug mode or you're not. Which is how it played out in my head, but as a library dev it makes total sense.